### ISSUE TITLE
hs._asm.guitk.menubar.legacy `nil` error

### ISSUE THREAD

**Issue by latenitefilms** [asmagill/hammerspoon_asm#13]
FYI:

```
2018-05-31 16:27:34: ********
2018-05-31 16:27:34: 16:27:34 ERROR:   LuaSkin: hs.eventtap callback error: ...mandPost/src/extensions/hs/_asm/guitk/menubar/legacy.lua:195: attempt to index a nil value (field '_statusitem')
stack traceback:
	...mandPost/src/extensions/hs/_asm/guitk/menubar/legacy.lua:195: in method 'setMenu'
	.../CommandPost/src/plugins/finalcutpro/viewer/overlays.lua:598: in function <.../CommandPost/src/plugins/finalcutpro/viewer/overlays.lua:590>
2018-05-31 16:27:34: ********
```

**Comment by latenitefilms** [comment]
Here's my workaround:

```lua
                mod._menu:returnToMenuBar()
                mod._menu:setMenu(menu):removeFromMenuBar()
                mod._menu:popupMenu(location, true)
```

It seems like the legacy code always expects the menubar menu to be in the system menubar?

**Comment by asmagill** [comment]
That's the one I just pushed a fix for and mentioned in the 2.0 roadmap
